dpppt-backend-sdk/dpppt-backend-sdk-ws/src/main/java/org/dpppt/backend/sdk/ws/controller/GaenController.java

```diff
@@ -158,6 +174,15 @@ public GaenController(GAENDataService dataService, FakeKeyService fakeKeyService
 		if (!nonFakeKeys.isEmpty()) {
 			dataService.upsertExposees(nonFakeKeys);
 		}
+		if (!nonFakeKeysDelayed.isEmpty()) {
+			// Hold back same day TEKs until 02:00 UTC of the next day (as RPIs are accepted by EN up to 2h after rolling period)
+			var tomorrowAt2AM = LocalDate.now(ZoneOffset.UTC)
```

Comment:
Again I'm not sure with the use of `now` here, as this might create wrong entries around midnight. As above, it would be better to check all the TEKs are in the past, then take the time of the latest TEK instead of `now`.

Reply:
Here, you're right, it might be better to  use the keyDate of the TEK as a basis. Nevertheless, the request would have to be timed quite well,  to actually make a  difference. Worst case that can happen is that one key is released a day too late.

I'll open an issue for that.